tenzir/tests/format_strings/lexer.tql

```diff
@@ -0,0 +1,28 @@
+// test: lexer
+
+"{}"
+
+f""
+
+f"{}"
+
+f"\""
+
+f"{ident}"
+
+f"{a+b}"
+
+f"pre {}"
+
+f"pre {} post"
+
+f"pre {"string"} post"
+
+f"{{}}"
+
+// TODO: Write more tests for this.
```

Comment:
Can you add the triple brace tests I shared in Slack, where we're formatting a record literal?